app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,15 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for :session, url: login_path, html: {class: "sign_in"} do |f|
```

Comment:
This should be definitely a `form_tag` not `form_for` - There is no `Session` model and `form_for` is meant and built for model-backed forms. `form_tag` serves exactly this purpose.